KarhooUISDK/CommonUI/AddressView/AddressView.swift

```diff
@@ -0,0 +1,235 @@
+//
+//  AddressView.swift
+//  KarhooUISDK
+//
+//  Created by Aleksander Wedrychowski on 16/11/2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import KarhooSDK
+import SwiftUI
+
+/// New Address view. Shows pick up, destination and places in between (if needed). View does not provide interactions by default.
+struct KarhooAddressView: View {
+
+    // MARK: - Nested types
+
+    enum Design {
+        /// Bordered view, with SDKs `background1` color and rounded corners.
+        case bordered
+        /// Bordered view, with SDKs `white` color and rounded corners.
+        case borderedWithWhiteBackground
+        /// Bordereless view with `white` color from SDK palette.
+        case `default`
+
+        var backgroundColor: UIColor {
+            switch self {
+            case .bordered:
+                return KarhooUI.colors.background1
+            case .borderedWithWhiteBackground, .default:
+                return KarhooUI.colors.white
```

Comment:
This shouldn't be white in dark mode. Let's use background 2. That's white in light mode and a dark gray in dark mode

Reply:
You are absolutely right. For some reason, I expected 'white' to be just a name that is changed into sort-of black in DM. In fact, it's not - it's only slightly darker shade of white. On my defense, in Figma `white` color was requested :D

Reply:
I know, no worries. We had a discussion with our designer last week on this exact topic and he updated the palette accordingly afterwards. We just need to use the new conventions. Bartek already handled updating the SDK and I cherrypicked the changes into the feature branch.